NOTICE.txt

```diff
@@ -210,8 +210,8 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 --------------------------------------------------------------------
 Dependency: github.com/elastic/apm-agent-go
-Version: v0.4.0
-Revision: a22839728ee5955c96524d56af60813250d26a5b
+Version: v0.5.0
```

Comment:
There was a quiet [v0.5.1](https://github.com/elastic/apm-agent-go/releases/tag/v0.5.1) release.

Reply:
missed that, thanks!